exporter/exporterhelper/README.md

```diff
@@ -17,12 +17,57 @@ The following configuration options can be modified:
 - `sending_queue`
   - `enabled` (default = true)
   - `num_consumers` (default = 10): Number of consumers that dequeue batches; ignored if `enabled` is `false`
-  - `queue_size` (default = 5000): Maximum number of batches kept in memory before data; ignored if `enabled` is `false`;
+  - `queue_size` (default = 5000): Maximum number of batches kept in memory or on disk (for persistent storage) before dropping; ignored if `enabled` is `false`
   User should calculate this as `num_seconds * requests_per_second` where:
     - `num_seconds` is the number of seconds to buffer in case of a backend outage
     - `requests_per_second` is the average number of requests per seconds.
+  - `persistent_storage_enabled` (default = false): When set, enables persistence via a file storage extension
 - `resource_to_telemetry_conversion`
   - `enabled` (default = false): If `enabled` is `true`, all the resource attributes will be converted to metric labels by default.
 - `timeout` (default = 5s): Time to wait per individual attempt to send data to a backend.
 
 The full list of settings exposed for this helper exporter are documented [here](factory.go).
+
+### Persistent Queue
+
+**Status: under development**
+
+When `persistent_storage_enabled` is set to true, the queue is being buffered to disk by the 
+[file storage extension](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/storage/filestorage). 
+It currently can be enabled only in OpenTelemetry Collector Contrib.
+
+
+```
```

Comment:
@pmm-sumo I am a big fan of your diagrams!

Reply:
Thank you! There is some complexity in how it works (which I was unable to simplify) so I thought the diagrams might be helpful in understanding what is going on here :)